drafter.gyp

```diff
@@ -41,17 +41,33 @@
         "src/RefractAST.h",
         "src/RefractAST.cc",
 
+        # librefract parts - will be separated into other project
+
         "src/refract/Element.h",
         "src/refract/Element.cc",
         "src/refract/Typelist.h",
         "src/refract/VisitableBy.h",
+
         "src/refract/Visitor.h",
+        "src/refract/Visitors.h",
+
         "src/refract/SerializeCompactVisitor.h",
         "src/refract/SerializeCompactVisitor.cc",
         "src/refract/SerializeVisitor.h",
         "src/refract/SerializeVisitor.cc",
         "src/refract/ComparableVisitor.h",
+        "src/refract/ComparableVisitor.cc",
+        "src/refract/TypeQueryVisitor.h",
+        "src/refract/TypeQueryVisitor.cc",
+        "src/refract/IsExpandableVisitor.h",
+        "src/refract/IsExpandableVisitor.cc",
+        "src/refract/ExpandVisitor.h",
```

Comment:
> expansion is not refract

@pksunkara Expansion is essentially described in Refract. And MSON Refract brings additional clarification to it. I do not see problem with what @klokane did here.